packages/manager/core/generator/app/conditional-templates/onboarding/index.tsx.hbs

```diff
@@ -1,49 +1,49 @@
-import React from "react";
-import { useTranslation } from "react-i18next";
-import { Card, OnboardingLayout } from "@ovhcloud/manager-components";
-import { useGuideUtils } from '@/hooks/useGuideUtils';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Card, OnboardingLayout } from '@ovhcloud/manager-components';
+import useGuideUtils from '@/hooks/guide/useGuideUtils';
 import Breadcrumb from '@/components/Breadcrumb/Breadcrumb';
-import onboardingImgSrc from "./onboarding-img.png";
+import onboardingImgSrc from './onboarding-img.png';
 
 export default function Onboarding() {
- const { t } = useTranslation('{{appName}}/onboarding');
- const link = useGuideUtils();
+  const { t } = useTranslation('onboarding');
```

Comment:
if we don't need appName anymore, we can just remove the .hbs extension, can't we ?

Reply:
👍